src/features/side_bar/components/AppNameAndIcon/AppNameAndIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import { ActionsBarBTN } from '@konturio/ui-kit';
+import { useAtom } from '@reatom/react';
+import { useEffect } from 'react';
+import app_config from '~core/app_config';
+import { i18n } from '~core/localization';
+import { currentAppPropertiesResourceAtom } from '~core/shared_state/currentApplication';
+
+type AppNameAndIconProps = {
+  wrapClassName?: string;
+  appNameClassName?: string;
+  isOpen: boolean;
+};
+
+export function AppNameAndIcon({
+  isOpen,
+  appNameClassName,
+  wrapClassName,
+}: AppNameAndIconProps) {
+  const [{ data: appParams }] = useAtom(currentAppPropertiesResourceAtom);
+  let iconPath = appParams?.sidebarIconUrl;
+
+  // trim beginning of url designed for builded apps that serve static files via '/active/static'
+  if (app_config.isDevBuild && iconPath?.startsWith('/active/static')) {
+    iconPath = iconPath.substring(14);
+  }
+
+  const appIcon = iconPath ? (
+    <img src={iconPath} width={24} height={24} alt={i18n.t('sidebar.icon_alt')} />
+  ) : null;
+
+  // change tab icon (not expected to work in Safari https://stackoverflow.com/questions/63781987/cant-change-favicon-with-javascript-in-safari)
+  useEffect(() => {
+    if (!iconPath) return;
+    const linkElements = document.querySelectorAll(
```

Comment:
try to understand what's going on here: 
so we have 4 links queried by this selector 
<img width="509" alt="Screenshot 2022-12-08 at 17 26 41" src="https://user-images.githubusercontent.com/20676525/206471887-a281131b-26f2-43f0-bebb-56e6747cb69
<img width="510" alt="Screenshot 2022-12-08 at 17 30 03" src="https://user-images.githubusercontent.com/20676525/206472601-03ba8c3a-dffc-4330-8cce-92e576828de8.png">
3.png">
then we replace every href with "/favicon/favicon.svg?v=2.0.22"
so in the end we have 
<img width="510" alt="Screenshot 2022-12-08 at 17 30 03" src="https://user-images.githubusercontent.com/20676525/206472692-558982e3-c7b2-418b-b8c6-2ae85d6316ff.png">
so why to keep public/favicon/favicon-16x16.png and public/favicon/favicon-32x32.png ?

Reply:
yeah.. good question